backend/app/api/routes/model_evaluation.py

```
@@ -0,0 +1,201 @@
+
+    Request:{ fine_tuning_ids: list[int] } (one or many).
+
+    Process:
+        For each ID, it fetches the fine-tuned model and its testing file from fine tuning table,
+        then queues a background task that runs predictions on the test set
+        and computes evaluation scores.
+
+    Response:
+        APIResponse with the created/active evaluation records and a success message.
+    """
+    client = get_openai_client(
```

Ruff (0.12.2)

Local variable `client` is assigned to but never used

Remove assignment to unused variable `client`

(F841)